docs/troubleshooting/FAQs.mdx

```diff
@@ -1,189 +1,166 @@
 ---
 id: faqs
 title: FAQs
-slug: /troubleshooting/
+slug: /troubleshooting/faqs
```

Comment:
This is going to break all of our historical links isn't it 😭

Maybe we need error code short URLs to make this easier huehuehue

Reply:
It shouldn't with this redirects config! I really hope it doesn't. That'd be a bug if it does.

https://github.com/typescript-eslint/typescript-eslint/blob/8427fbffc89791d31b9032395e2d01a29c974f48/packages/website/docusaurus.config.mts#L290-L293

But yeah strong +1 to error code short URLs.